posix.mak

```diff
@@ -375,6 +377,16 @@ unittest/%.run : $(ROOT)/unittest/test_runner
 %.test : $(LIB)
 	$(MAKE) -f $(MAKEFILE) $(addsuffix .test,$(patsubst %.d,%,$(wildcard $*/*)))
 
+%.debug_with_debugger : %.d $(LIB)
+	$(DMD) $(DFLAGS) -main -unittest $(LIB) -defaultlib= -debuglib= $(LINKDL) $<
```

Comment:
because the target needs to know what file it should run in the debugger, otherwise you pass the debugger nothing to debug